<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newPaciente_paciente_ui</class>
 <widget class="QWidget" name="newPaciente_paciente_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>548</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>548</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>548</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Paciente</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rc.qrc">
    <normaloff>:/new/Add-Male-User.png</normaloff>:/new/Add-Male-User.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <property name="verticalSpacing">
      <number>6</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Apellido Paterno :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_FatherName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="0" column="5" colspan="3">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Apellido Materno :</string>
       </property>
      </widget>
     </item>
     <item row="0" column="8" colspan="2">
      <widget class="QLineEdit" name="lineEdit_MotherName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Primer Nombre :</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_FirstName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="5" colspan="3">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Segundo Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="8" colspan="2">
      <widget class="QLineEdit" name="lineEdit_SecondName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Dni :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_Dni">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="5">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Ocupación :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="8" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Ocupation">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Teléfono</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_Telephone">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="3" column="5">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Celular :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="8" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Cellphone">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Email :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="9">
      <widget class="QLineEdit" name="lineEdit_Email">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="10">
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Pais :                    </string>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_Country">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="5" colspan="3">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Departamento :   </string>
       </property>
      </widget>
     </item>
     <item row="6" column="8" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Departamento">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Provincia :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEdit_Provincia">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="7" column="5">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Distrito :</string>
       </property>
      </widget>
     </item>
     <item row="7" column="8" colspan="2">
      <widget class="QLineEdit" name="lineEdit_Distrito">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>Lugar :</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="9">
      <widget class="QLineEdit" name="lineEdit_Lugar"/>
     </item>
     <item row="9" column="0" colspan="10">
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="10" column="0" colspan="2">
      <widget class="QLabel" name="label_20">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Fecha de Nacimiento :</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2" colspan="2">
      <widget class="QDateEdit" name="dateEdit_DateofBirth">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentSection">
        <enum>QDateTimeEdit::YearSection</enum>
       </property>
       <property name="displayFormat">
        <string>yyyy/MM/dd</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="11" column="0" colspan="2">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Donación de Organos :</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2" colspan="2">
      <widget class="QComboBox" name="comboBox_OrganDonor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>No información</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Si</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="11" column="7" colspan="3">
      <widget class="QComboBox" name="comboBox_Sex">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Femenino</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Masculino</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Cirujías Previas :</string>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="QComboBox" name="comboBox_PrefSurgery">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Si</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Medicación Actual :</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QComboBox" name="comboBox_CurrentMedication">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Si</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>Alergias :</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2" colspan="8">
      <widget class="QLineEdit" name="lineEdit_alergias"/>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Antecedentes :</string>
       </property>
      </widget>
     </item>
     <item row="15" column="2" colspan="8">
      <widget class="QLineEdit" name="lineEdit_antecedentes"/>
     </item>
     <item row="16" column="0" colspan="10">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="QLabel" name="label_18">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Persona a recurrir :</string>
       </property>
      </widget>
     </item>
     <item row="17" column="3" colspan="7">
      <widget class="QLineEdit" name="lineEdit_TutorPerson">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="18" column="0" colspan="3">
      <widget class="QLabel" name="label_12">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Teléfonos de persona a recurrir :</string>
       </property>
      </widget>
     </item>
     <item row="18" column="3" colspan="7">
      <widget class="QLineEdit" name="lineEdit_TelephoneTutorPerson">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="19" column="0" rowspan="3" colspan="10">
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="20" column="9" rowspan="3">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="label_22">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Historia Clínica :</string>
       </property>
      </widget>
     </item>
     <item row="10" column="7" colspan="3">
      <widget class="QComboBox" name="comboBox_MaritalStatus">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Soltero/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Casado/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Divorciado/a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Viudo/a</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="10" column="6">
      <widget class="QLabel" name="label_21">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Estado Civil :</string>
       </property>
      </widget>
     </item>
     <item row="11" column="6">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Sexo :</string>
       </property>
      </widget>
     </item>
     <item row="22" column="2" colspan="3">
      <widget class="QLineEdit" name="lineEdit_historiaClinica">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="12" column="3" colspan="7">
      <widget class="QLineEdit" name="lineEdit_cirujias_previas"/>
     </item>
     <item row="13" column="3" colspan="7">
      <widget class="QLineEdit" name="lineEdit_Medicacion_actual"/>
     </item>
     <item row="2" column="4">
      <widget class="QLineEdit" name="lineEdit_dni_to_insert">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>15</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="22" column="5">
      <widget class="QCheckBox" name="checkBox_Historia">
       <property name="text">
        <string>No Historia</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>22</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_Acept">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_Save">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="pushButton_Cancel">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEdit_FatherName</tabstop>
  <tabstop>lineEdit_MotherName</tabstop>
  <tabstop>lineEdit_FirstName</tabstop>
  <tabstop>lineEdit_SecondName</tabstop>
  <tabstop>lineEdit_Dni</tabstop>
  <tabstop>lineEdit_Ocupation</tabstop>
  <tabstop>lineEdit_Telephone</tabstop>
  <tabstop>lineEdit_Cellphone</tabstop>
  <tabstop>lineEdit_Email</tabstop>
  <tabstop>lineEdit_Country</tabstop>
  <tabstop>lineEdit_Departamento</tabstop>
  <tabstop>lineEdit_Provincia</tabstop>
  <tabstop>lineEdit_Distrito</tabstop>
  <tabstop>lineEdit_Lugar</tabstop>
  <tabstop>dateEdit_DateofBirth</tabstop>
  <tabstop>comboBox_MaritalStatus</tabstop>
  <tabstop>comboBox_OrganDonor</tabstop>
  <tabstop>comboBox_Sex</tabstop>
  <tabstop>comboBox_PrefSurgery</tabstop>
  <tabstop>lineEdit_cirujias_previas</tabstop>
  <tabstop>comboBox_CurrentMedication</tabstop>
  <tabstop>lineEdit_Medicacion_actual</tabstop>
  <tabstop>lineEdit_alergias</tabstop>
  <tabstop>lineEdit_antecedentes</tabstop>
  <tabstop>lineEdit_TutorPerson</tabstop>
  <tabstop>lineEdit_TelephoneTutorPerson</tabstop>
  <tabstop>lineEdit_historiaClinica</tabstop>
  <tabstop>pushButton_Acept</tabstop>
  <tabstop>pushButton_Save</tabstop>
  <tabstop>pushButton_Cancel</tabstop>
  <tabstop>lineEdit_dni_to_insert</tabstop>
 </tabstops>
 <resources>
  <include location="rc.qrc"/>
 </resources>
 <connections/>
</ui>
